src/components/FormFields/SelectField/SelectField.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Field } from 'formik';
+import classnames from 'classnames';
+import { string } from 'prop-types';
+
+import { Label } from '../../Label';
+import { Select } from '../../Select';
+
+export const SelectField = ({
+  id,
+  name,
+  placeholder,
+  label,
+  className,
+  ...props
+}) => {
+  return (
+    <div className={classnames('select-field--wrapper', className)}>
+      {!!label && <Label label={label} />}
```

Comment:
`!!label` -> `label` No need to use `!!`

Reply:
fixed in next PR